src/main/java/seedu/duke/command/add/AddTaskCommand.java

```diff
@@ -17,6 +17,11 @@
     private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d-M-yyyy HHmm");
     private String desc;
     private LocalDateTime dateTimeOfDeadline;
+    public static final String FORMAT = COMMAND_WORD + " -t <task name> [-by <deadline>]";
+    public static final String HELP =   "Add a task to the scheduler." +
+                                        "\nFormat: " + FORMAT +
+                                        "\nExample usage: add -t Read Book" +
+                                        "\n               add -t Return Book -by 2-2-2020 1800\n\n";
```

Comment:
An example like `30-12-2020` will probably be better since `2-2` is a bit ambiguous about month and day.